napari/_vispy/visuals/labels.py

```diff
@@ -0,0 +1,36 @@
+from vispy.scene.visuals import create_visual_node
+from vispy.visuals.image import ImageVisual
+from vispy.visuals.shaders import Function, FunctionChain
+
+from napari._vispy.visuals.util import TextureMixin
+
+SCALE_R8 = 'float cmap(float v) { return v*255; }'
+SCALE_R16 = 'float cmap(float v) { return v*65535; }'
+
+
+class LabelVisual(TextureMixin, ImageVisual):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
```suggestion
```

Do you have to have this impl that just forward arguments, can we just remove it ? Or is there an issue with multiple inheritance?

Reply:
Nope. This is an artifact from coding before I implemented TextureMixin. I will fix this evening.